pkg/util/cmapichecker/cmapichecker.go

```diff
@@ -81,11 +77,13 @@ func New(restcfg *rest.Config, scheme *runtime.Scheme, namespace string) (Interf
 	}, nil
 }
 
-// Check attempts to perform a dry-run create of a cert-manager *v1alpha2*
+// Check attempts to perform a dry-run create of a cert-manager
 // Certificate resource in order to verify that CRDs are installed and all the
 // required webhooks are reachable by the K8S API server.
-// We use v1alpha2 API to ensure that the API server has also connected to the
-// cert-manager conversion webhook.
+// Originally we used the v1alpha2 API to ensure that the API server has also
```

Comment:
Thanks for this comment. In a theoretical future we could check against defaulting functionality if we add it.